autopush/tests/test_endpoint.py

```diff
@@ -173,7 +170,7 @@ def test_update_too_much_data(self):
 
         def handle_finish(result):
             self.assertTrue(result)
-            self.status_mock.assert_called_with(401)
+            self.status_mock.assert_called_with(413)
```

Comment:
I considered that, but it seems like it'd create a huge amount of work to do any tweak later. I did add a test to make sure the error has JSON. I can have it verify the rest of the attributes too, as long as it works once, since all the others call the same function I don't see why it wouldn't be fine for them.